package.json

```
@@ -54,32 +51,29 @@
     "@testing-library/dom": "^8.1.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.6.2",
-    "@babel/core": "^7.6.2",
-    "@babel/plugin-transform-modules-commonjs": "^7.6.0",
-    "@babel/preset-env": "^7.6.2",
     "@commitlint/cli": "^13.1.0",
     "@commitlint/config-conventional": "^13.1.0",
-    "@testing-library/jest-dom": "^5.0.2",
-    "@types/jest": "^27.0.0",
+    "@sveltejs/vite-plugin-svelte": "^2.4.1",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@vitest/coverage-c8": "^0.32.0",
     "all-contributors-cli": "^6.9.0",
-    "babel-eslint": "^10.0.3",
-    "babel-jest": "^27.0.6",
     "doctoc": "^2.0.0",
-    "eslint": "^7.2.0",
-    "eslint-config-standard": "^16.0.0",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-node": "^11.0.0",
-    "eslint-plugin-promise": "^5.1.0",
-    "eslint-plugin-simple-import-sort": "^7.0.0",
-    "eslint-plugin-svelte3": "^3.0.0",
+    "eslint": "^8.42.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^16.0.0",
+    "eslint-plugin-promise": "^6.1.1",
+    "eslint-plugin-simple-import-sort": "^10.0.0",
+    "eslint-plugin-svelte": "^2.30.0",
```

the standard plugin isn't in alphabetical order
```suggestion
    "eslint-plugin-import": "^2.27.5",
    "eslint-plugin-n": "^16.0.0",
    "eslint-plugin-promise": "^6.1.1",
    "eslint-plugin-simple-import-sort": "^10.0.0",
    "eslint-config-standard": "^17.1.0",
    "eslint-plugin-svelte": "^2.30.0",
```